treeherder/etl/runnable_jobs.py

```diff
@@ -18,37 +18,40 @@ def _taskcluster_runnable_jobs(project):
     if not decision_task_id:
         return []
 
-    tc_graph_url = RUNNABLE_JOBS_URL.format(task_id=decision_task_id)
-    validate = URLValidator()
-    try:
-        validate(tc_graph_url)
-    except ValidationError:
-        logger.warning('Failed to validate %s', tc_graph_url)
-        return []
+    for run_number in range(0, 10):
```

Comment:
run_number is not higher than 5 (max 5 retries).

Reply:
Yeah, that makes sense.  Thanks @Archaeopteryx :)